utils/config.ts

```diff
@@ -58,8 +62,8 @@ export type FastLoginClientConfig = t.TypeOf<typeof FastLoginClientConfig>;
 
 // Fast Login Client Configuration
 export const FunctionsAppClientConfig = t.type({
-  FUNCTIONS_APP_CLIENT_BASE_URL: NonEmptyString,
-  FUNCTIONS_APP_SUBSCRIPTION_KEY: NonEmptyString
```

Comment:
There are two other occurrences of `FUNCTIONS_APP_SUBSCRIPTION_KEY` to change (README and config.mocks.ts).